src/db/api/beacon/beacon.ts

```diff
@@ -30,7 +30,7 @@ export class BeaconDB extends DatabaseService implements IBeaconDb {
 
   public async getState(): Promise<BeaconState> {
     const buf = await this.db.get(encodeKey(Bucket.chainInfo, Key.state));
-    return deserialize(buf, BeaconState);
+    return deserialize(buf, BeaconState) as unknown as BeaconState;
```

Comment:
what's going on here?

Reply:
I think this is because the return type of `deserialize` changed from `any` to `SerializableValue`.
The ssz types had not been showing recently, but when I merged in master, it showed all of these issues.

Perhaps we should change the return type back to `any` and cut a new release?

Reply:
yeah probably

Reply:
I'll cut a new release of ssz after https://github.com/ChainSafe/ssz-js/pull/95 and include it in this PR